README.md

```diff
@@ -1,70 +1,208 @@
+- LinkedIn: [![ayodeji-abidemi](https://img.shields.io/badge/-AyodejiAbidemi-white?logo=LinkedIn&logoColor=181717&style=plastic)](https://linkedin.com/in/ayodeji-abidemi)
+
+👤 **Mohamed Hassan Noor**
+
+- GitHub: [![MohamedHNoor](https://img.shields.io/badge/-MohamedHNoor-white?logo=GitHub&logoColor=181717&style=plastic)](https://github.com/MohamedHNoor)
+- Twitter: [![MohamedHNoor](https://img.shields.io/badge/-MohamedHNoor-blue?logo=Twitter&logoColor=skyBlue&style=plastic)](https://twitter.com/MohamedHNoor)
+- LinkedIn: [![MohamedHNoor](https://img.shields.io/badge/-MohamedHNoor-white?logo=LinkedIn&logoColor=181717&style=plastic)](https://www.linkedin.com/in/mohamedhnoor/)
+
+👤 **Houda Mzari**
+
+- GitHub: [![houdamzari](https://img.shields.io/badge/-houdamzari-white?logo=GitHub&logoColor=181717&style=plastic)](https://github.com/houdamzari/)
+- LinkedIn: [![houdamzari](https://img.shields.io/badge/-houdamzari-white?logo=LinkedIn&logoColor=181717&style=plastic)](https://www.linkedin.com/in/houda-mzari/)
+
+👤 **Kashif Saleem**
+
+- GitHub:[![Kashif](https://img.shields.io/badge/-Kashif-white?logo=GitHub&logoColor=181717&style=plastic)](https://github.com/Kashif-Saleem-Ghuman)
+- Twitter:[![Kashif](https://img.shields.io/badge/-Kashif-blue?logo=Twitter&logoColor=skyBlue&style=plastic)](https://twitter.com/Kashif14Saleem)
+- LinkedIn:[![Kashif](https://img.shields.io/badge/-Kashif-white?logo=LinkedIn&logoColor=181717&style=plastic)](https://www.linkedin.com/in/kashifsaleemghuman/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Allow users to choose a language for the application**
+- [ ] **Add payment option**
+- [ ] **Some UX improvements: include transitions and/or animations, etc.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/houdamzari/Cars-Rental-FrontEnd/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please leave a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- We would like to thank Microverse for providing the reading materials that aided us to during the project development.
+- We would like to give Credit to 🙏 Murat Korkmaz the author of the original Vespa- Responsive Redesign, as required by the Creative Commons License.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./License) licensed.
```

Comment:
- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪